### ISSUE TITLE
git lab calendar fails with `list index out of range`

### ISSUE THREAD

**Issue by mattiasb** [seveas/git-spindle#124]
Running `git lab calendar` after having added an account in #123 gives me a `IndexError: list index out of range` error. Like this:

```
~ $ git lab calendar 
  Oct     Nov       Dec     Jan       Feb     Mar     Apr     May       Jun     Jul       Aug     Sep     Oct 
Traceback (most recent call last):
  File "/home/mattiasb/.local/bin/git-lab", line 11, in <module>
    GitLab().main()
  File "/home/mattiasb/.local/lib/python3.5/site-packages/gitspindle/__init__.py", line 344, in main
    func(opts)
  File "/home/mattiasb/.local/lib/python3.5/site-packages/gitspindle/gitlab.py", line 300, in calendar
    p5  = commits[int(round(len(commits) * 0.95))]
IndexError: list index out of range
```

**Comment by seveas** [comment]
Fixed in 12fff11

**Comment by mattiasb** [comment]
Just tried this on [one of my repos](https://gitlab.com/mattiasb/git-lab.rs) but got this:
```
 mattiasb   master  ~  Code  Projects  git-lab.rs  git lab calendar 
  Sep Oct       Nov     Dec     Jan       Feb     Mar     Apr       May     Jun     Jul       Aug     Sep   
Traceback (most recent call last):
  File "/home/mattiasb/.local/bin/git-lab", line 11, in <module>
    GitLab().main()
  File "/home/mattiasb/.local/lib/python3.6/site-packages/gitspindle/__init__.py", line 372, in main
    func(opts)
  File "/home/mattiasb/.local/lib/python3.6/site-packages/gitspindle/gitlab.py", line 307, in calendar
    p5  = commits[int(round(len(commits) * 0.95))]
IndexError: list index out of range
 ```

**Comment by seveas** [comment]
You need to do more on gitlab :-)
Fixed in 6a6c920

**Comment by mattiasb** [comment]
Hehe. Awesome thanks!
